src/trigger/text_area.cpp

```diff
@@ -25,6 +25,11 @@
 #include "video/drawing_context.hpp"
 #include "video/layer.hpp"
 
+namespace
+{
+std::string anchor_points[] = { "middle", "topleft", "top", "topright", "left", "right", "bottomleft", "bottom", "bottomright" };
+}
+
```

Comment:
~~Is there a reason to put it in `namespace {}`? Also, it can be made `static` if it's used only in this cpp file, and it can be marked as `const` if it isn't edited at runtime~~

I believe you won't need this anyways, given the possible changes below.